modules/react/popup/stories/examples/CustomTarget.tsx

```diff
@@ -13,9 +13,9 @@ interface MyTargetProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   label: string;
 }
 
-const MyTarget = ({label, ...props}: MyTargetProps) => {
-  return <button {...props}>{label}</button>;
-};
+const MyTarget = React.forwardRef(({label, ...props}: MyTargetProps, ref) => {
+  return <button {...props} ref={ref}>{label}</button>;
+});
```

Comment:
Type error dealing with the forwarded ref...

```suggestion
const MyTarget = React.forwardRef<HTMLButtonElement, MyTargetProps>(({label, ...props}, ref) => {
  return (
    <button {...props} ref={ref}>
      {label}
    </button>
  );
});
```